docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -0,0 +1,402 @@
+# Machine Learning Training
+
+Machine Learning Training allows to construct, control and train a machine learning model in Tizen devices.
+
+Please note that:
+
+* This feature remains experimental for Tizen 6.0 and is not full-fledged yet for every needs.
+* Every example code is not fully handling error for brevity. Error should be handled more extensively then example code written here.
+
+## Prerequisites
+
+To enable your application to use the machine learning training functionality:
```

Comment:
follow as per https://github.com/zhoonit/tizen-docs/pull/1/files#r478799057

Reply:
Did you mean this? 
```suggestion
To enable your application to use the machine learning training functionality, follow as per:
```

Reply:
I mean machine learning training `functionality` -> as per decided from https://github.com/zhoonit/tizen-docs/pull/1/files#r478799057

Reply:
Changed to API